ci/cron/daily-compat.yml

```diff
@@ -80,3 +80,49 @@ jobs:
                $(Slack.team-daml)
         displayName: report
         condition: always()
+  - job: performance_report
+    timeoutInMinutes: 120
+    pool:
+      name: "linux-pool"
+    steps:
+      - checkout: self
+      - bash: ci/dev-env-install.sh
+        displayName: 'Build/Install the Developer Environment'
+      - bash: ci/configure-bazel.sh
+        displayName: 'Configure Bazel for root workspace'
+        env:
+          IS_FORK: $(System.PullRequest.IsFork)
+          # to upload to the bazel cache
+          GOOGLE_APPLICATION_CREDENTIALS_CONTENT: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)
+      - bash: |
+          set -euo pipefail
+          eval "$(dev-env/bin/dade assist)"
+          BASELINE="cebc26af88efef4a7c81c62b0c14353f829b755e"
+          OUT="$(Build.StagingDirectory)/perf-results.json"
+          if git diff --exit-code $BASELINE -- daml-lf/scenario-interpreter/src/perf >&2; then
+              # no changes, all good
+              ci/cron/compare-perf.sh $BASELINE > $OUT
+          else
+              # the tests have changed, we need to figure out what to do with
```

Comment:
One option would be to always use the benchmark code from HEAD but against different versions of damlc and the interpreter. Changing damlc is straightforward but since the interpreter is used as a library this is going to be a bit more tricky.

Reply:
An alternative I played with was to checkout the `daml-lf/scenario-runner/src/perf` folder as of master on the baseline, which would test the current benchmark on the baseline damlc/speedy combination. But that doesn't work for arbitrary past commits (i.e. the perf folder depends on the rest of the repo so might not compile if interfaces have changed), so I eventually decided that, at least for the next time the benchmark is changed, we'd have to look at it manually and decide whether that's a viable way forward.